packages/remirror__extension-link/src/link-extension.ts

```diff
@@ -145,10 +149,41 @@ export interface LinkOptions {
    * value.
    *
    * @default
-   * /(?:(?:(?:https?|ftp):)?\/\/)?(?:\S+(?::\S*)?@)?(?:(?:[\da-z\u00A1-\uFFFF][\w\u00A1-\uFFFF-]{0,62})?[\da-z\u00A1-\uFFFF]\.)+[a-z\u00A1-\uFFFF]{2,}\.?(?::\d{2,5})?(?:[#/?]\S*)?/gi
+   * /(?:(?:(?:https?|ftp):)?\/\/)?(?:\S+(?::\S*)?@)?(?:(?:[\da-z\u00A1-\uFFFF][\w\u00A1-\uFFFF-]{0,62})?[\da-z\u00A1-\uFFFF]\.)*(?:(?:\d(?!\.)|[a-z\u00A1-\uFFFF])(?:[\da-z\u00A1-\uFFFF][\w\u00A1-\uFFFF-]{0,62})?[\da-z\u00A1-\uFFFF]\.)+[a-z\u00A1-\uFFFF]{2,}(?::\d{2,5})?(?:[#/?]\S*)?/gi
    */
   autoLinkRegex?: Static<RegExp>;
 
+  /**
+   * An array of valid Top Level Domains (TLDs) to limit the scope of auto linking.
+   *
+   * @remarks
+   *
+   * The default autoLinkRegex does not limit the TLD of a URL for performance and maintenance reasons.
+   * This can lead to the auto link behaviour being overly aggressive.
+   *
+   * Remirror uses https://www.npmjs.com/package/tlds to provide aa list of all valid TLDs.
```

Comment:
Typo

```suggestion
   * Remirror uses https://www.npmjs.com/package/tlds to provide a list of all valid TLDs.
```

Reply:
This is out of date anyway. Will remove this.